api/db/init.ts

```diff
@@ -0,0 +1,36 @@
+import sequelize from './connect';
+import UserService from '../secvices/user.service';
+
+const init = async () => {
+    await sequelize.sync({
+        force: true
+    });
+
+    await UserService.create({
```

Comment:
Actually you can use seeds for such database initial data filling in. Sequelize has such features:
https://sequelize.org/master/manual/migrations.html
(scroll to section about Seeds).

Reply:
done